### PR TITLE
LPS-106619 Analytics Reports sidebar panel

### PR DESCRIPTION
**Pull request by dgarciasarai**
CI here https://github.com/dgarciasarai/liferay-portal/pull/151
Thanks!

### CODE REVIEW

**Comment by ealonso** [comment]
Just started reviewing :) <br><br>:octocat: *Sent from [GH](http://nodegh.io).*

**Comment by ealonso** [comment]
@dgarciasarai puedes separar 83ca80d en tres commits? uno con el Impl, otro con el auto y otro con el packageinfo

**Comment by dgarciasarai** [comment]
👍

File: modules/dxp/apps/analytics/analytics-reports-web/src/main/resources/content/Language.properties [hunk 1]
```diff
@@ -0,0 +1,3 @@
+#TODO keys as content-performace???
```
Comment (dgarciasarai) [inline modules/dxp/apps/analytics/analytics-reports-web/src/main/resources/content/Language.properties]: Remove this 😬

File: modules/dxp/apps/analytics/analytics-reports-web/src/main/resources/content/Language.properties [hunk 2]
```diff
@@ -0,0 +1,3 @@
...
+analytics-reports=Content Performance
```
Comment (dgarciasarai) [inline modules/dxp/apps/analytics/analytics-reports-web/src/main/resources/content/Language.properties]: `content-performance` or `analytics-reports`?

File: modules/dxp/apps/analytics/analytics-reports-web/src/main/resources/content/Language_ar.properties [hunk 3]
```diff
@@ -0,0 +1,2 @@
+analytics-reports=Content Performance (Automatic Copy)
```
Comment (dgarciasarai) [inline modules/dxp/apps/analytics/analytics-reports-web/src/main/resources/content/Language_ar.properties]: Rebuild lang if the keys change

File: modules/dxp/apps/analytics/analytics-reports-web/src/main/resources/content/Language.properties (hunk 2 above)
Comment (dgarciasarai) [reply modules/dxp/apps/analytics/analytics-reports-web/src/main/resources/content/Language.properties]: Change to `content-performance`

**Comment by darquesdev** [comment]
Just started reviewing :) <br><br>:octocat: *Sent from [GH](http://nodegh.io).*

**Comment by darquesdev** [comment]
@ealonso you can close this PR

**Pull request closed without merging**